backend/src/main/java/com/woowacourse/kkogkkog/config/WebClientConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.woowacourse.kkogkkog.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
+public class WebClientConfig {
+
+    @Bean
+    public WebClient webClient() {
+        return WebClient
+            .builder()
+            .build();
+    }
```

Comment:
builder 호출 이외의 생성 방법은 없을려나요?

```suggestion
    @Bean
    public WebClient webClient() {
        return WebClient.builder().build();
    }
```

Reply:
```suggestion
    @Bean
    public WebClient webClient() {
        return WebClient.create()
    }
```
create 메서드로 변경하였습니다. 다른 좋은 방법이 있을까요?